keyboards/ut472/ut472.h

```diff
@@ -32,3 +32,4 @@
   }
 
 
+#define LAYOUT_ut472 LAYOUT
```

Comment:
What's the reason for adding this instead of simply using `LAYOUT`?

Reply:
Using `LAYOUT` early on was failing during compiling. Something else must have been broken though, because I just removed it and it works now.